### DISCUSSION TITLE
Pre-compiled dxgi.dll

### DISCUSSION THREAD

**Discussion by Laserstrahl** [valinet/ExplorerPatcher#351] [Ideas]
Any chance to provide the already compiled dxgi.dll for the latest releases instead of the installer? Since when you already have dxgi.dll copied under windows, the installer / setup-routine asked to uninstall, I just want to overwritte to "update". Would be great! :) Cheers

**Comment by valinet** [comment]
Seems I have to stick this somewhere as well:

https://github.com/valinet/ExplorerPatcher/issues/345#issuecomment-968349676

~~I will think of a more obvious solution for this as well, adding to README, idk...~~ No, I will add a note to the release notes.

Thanks

**Reply by Laserstrahl** [reply]
Ah well thx! Yeah, would be great if you could add it to the release notes for future reference :)

**Reply by valinet** [reply]
I will, when I push the next update. Thx
